qemu/tests/cfg/hpt_max_page_size.cfg

```diff
@@ -26,3 +26,10 @@
                     disable_kvm = yes
                     auto_cpu_model = yes
                     login_timeout = 1800
+       - 16M: # Please set 1G huge page as default huge page size on power9
```

Comment:
Actually, the test case will fail if there's not 1G huge page as the default on host. But I will still add a check method as a reminder, thanks.

Reply:
Updated

Reply:
Actually, If they all belong to "16M", they should be placed in a subset of 16M instead of side by side.
And the command parameters should be displayed more clearly, such as: set_hugepages_cmd
Like below:

```
        - 16m:
            expected_value = "16384"
            machine_type_extra_params += ",cap-hpt-max-page-size=16M"
            variants:
                - tcg_mode: # setup hugepages but without mem backing file
                    setup_hugepages = yes
                    machine_type_extra_params += ",accel=tcg"
                    disable_kvm = yes
                    auto_cpu_model = yes
                    login_timeout = 1800
                - hugepage_reset: # Please set 1G huge page as default huge page size on power9
                    only with_hugepage
                    expected_hugepage_size = "1048576"
                    origin_nr = 1
                    set_hugepages_cmd = "echo ${origin_nr} > /proc/sys/vm/nr_hugepages"
                    get_hugepages_num_cmd = "grep HugePages_Total /proc/meminfo"
```

Reply:
You have your points, and I will have this in the patch, thanks.